### DISCUSSION TITLE
How can I change the LSP hover background color?

### DISCUSSION THREAD

**Discussion by ghost** [VonHeikemen/lsp-zero.nvim#367] [Q&A] [answered]
So, I want the **LSP hover** background color to be the same as the **Autocomplete popup** background color.

| LSP hover | Autocomplete popup |
| :---: | :---: |
| ![image](https://github.com/VonHeikemen/lsp-zero.nvim/assets/139251819/63435891-60f5-4c55-b12e-639ca6dddc88) | ![image](https://github.com/VonHeikemen/lsp-zero.nvim/assets/139251819/f629991d-52f8-44bd-b0dc-9cef0b041e79) |

Will it be possible to do it? If so, how can I do it?

Thank you in advance!

**Comment by VonHeikemen** [comment, marked as answer]
Modify the "highlight group" called `NormalFloat`.

You can link it to an existing group. I don't know what highlight group nvim-cmp uses in that window. In the documentation they mention one called `CmpItemMenu`, maybe that works.

```lua
vim.api.nvim_set_hl(0, 'NormalFloat', {link = 'CmpItemMenu'})
```

Or you can set the colors for the background.

```lua
vim.api.nvim_set_hl(0, 'NormalFloat', {guibg='#1e1e2e'})
```

**Reply by ghost** [reply]
Oh! So it was just the normal float highlight group. Thank you!
